src/www/ui/api/documentation/openapi.yaml

```diff
@@ -866,7 +866,51 @@ paths:
                   $ref: '#/components/schemas/Info'
         default:
           $ref: '#/components/responses/defaultResponse'
-
+  /uploads/{id}/uploadtree/{itemId}/view:
```

Comment:
As discussed, would you like to change the `/uploadtree` to `/item`? to be consistent?

Reply:
Done @GMishx